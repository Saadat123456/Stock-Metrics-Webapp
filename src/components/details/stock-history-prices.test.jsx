import { render } from '@testing-library/react';
import StockHistoryPrices from './stock-history-prices';

test('Renders Correctly', () => {
  const data = [
    [
      '1650043440',
      40493.35613391777,
    ],
    [
      '1650043740',
      40502.17406780077,
    ],
    [
      '1650044040',
      40524.388687130406,
    ],
    [
      '1650044340',
      40457.363542248575,
    ],
    [
      '1650044640',
      40425.773716666015,
    ],
    [
      '1650044940',
      40422.37122513517,
    ],
  ];

  const { container } = render(
    <StockHistoryPrices description={data} />,
  );
  expect(container.firstChild).toMatchSnapshot();
});
