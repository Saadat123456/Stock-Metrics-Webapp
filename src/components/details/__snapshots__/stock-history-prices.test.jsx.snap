// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Correctly 1`] = `
<div
  class="col"
>
  <h3
    class="row heading"
  >
    Daily Prices
  </h3>
  <div
    class="historyContainer"
  >
    <div
      class="row historyPrices"
    >
      <div
        class="price"
      >
        40493.35613391777
      </div>
      <div
        class="dateTime"
      >
        22:24
         | 
        05/03/22
      </div>
    </div>
    <div
      class="row historyPrices"
    >
      <div
        class="price"
      >
        40502.17406780077
      </div>
      <div
        class="dateTime"
      >
        22:29
         | 
        05/03/22
      </div>
    </div>
    <div
      class="row historyPrices"
    >
      <div
        class="price"
      >
        40524.388687130406
      </div>
      <div
        class="dateTime"
      >
        22:34
         | 
        05/03/22
      </div>
    </div>
    <div
      class="row historyPrices"
    >
      <div
        class="price"
      >
        40457.363542248575
      </div>
      <div
        class="dateTime"
      >
        22:39
         | 
        05/03/22
      </div>
    </div>
    <div
      class="row historyPrices"
    >
      <div
        class="price"
      >
        40425.773716666015
      </div>
      <div
        class="dateTime"
      >
        22:44
         | 
        05/03/22
      </div>
    </div>
    <div
      class="row historyPrices"
    >
      <div
        class="price"
      >
        40422.37122513517
      </div>
      <div
        class="dateTime"
      >
        22:49
         | 
        05/03/22
      </div>
    </div>
  </div>
</div>
`;
